tests/testthat/test-dplyr_methods.R

```diff
@@ -318,3 +318,18 @@ test_that("rowwise()", {
     expect_equal(dim(fd), c(ncol(df), 1))
     expect_identical(fd[[1]], sapply(df$lys, sum))
 })
+
+test_that("group_split()", {
+    fd <- df |> 
+    group_split("groups")
+    expect_equal(length(fd), length(unique(df$groups)))
+    
+    fd <- df |> 
+    group_split("vst.variable")
```

Comment:
One question: what is `vst.variable`? I cannot find it in the object metadata.

Reply:
It's in the rowData. I realise now that code wasn't doing what I thought it did and that it wouldn't have been helpful if it had!